### DISCUSSION TITLE
Is transparency for the kitty background supported for i3?

### DISCUSSION THREAD

**Discussion by Danielgb23** [kovidgoyal/kitty#4169] [Q&A] [answered]
Because it's not working. In google it was said you need compton but it's not the same thing because it makes the letters/icat pics transparent too

**Comment by kovidgoyal** [comment, marked as answer]
no it isnt

**Comment by trygveaa** [comment]
Should work fine with i3 if you run a compositor like compton or picom as well. Don't know why you get transparent letters/images, maybe wrong config?

**Reply by Danielgb23** [reply]
It's kitty. It works with urxvt but not with kitty.

**Reply by interdisziplina** [reply]
For me, it works great using `compton`.
